dotnet/src/SemanticKernel.Abstractions/Diagnostics/ITelemetryService.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Diagnostics;
```

Comment:
@IanNorris I've submitted a PR against your repo to resolve the merge conflicts with our latest changes. If you can merge that PR then we should be able to get this completed.

Reply:
Approved, thanks!

Reply:
Looking at the change list on this PR now, it looks very inflated. Hopefully the merge will go through cleanly though.